jenkins/shared-library/vars/dockerHelper.groovy

```diff
@@ -0,0 +1,78 @@
+void updateDockerImage(String targetImageName, String sourceImageName, String architecture) {
```

Comment:
looks good, but what changed that we need this additional logic now?

Reply:
This function is to create the multi-architecture image.
ofc, docker support for this is weak and not well design, imo. 😅 
The docker buildx command will append to an existing image but it doesnt override the current platform if it exists already.  So have to manually create the image in certain case.

Reply:
consider adding comment to that effect in the code